### PR TITLE
New package: kwallet-cli-3.02

### PR DESCRIPTION
**Pull request by DBLouis**

### CODE REVIEW

File: srcpkgs/kwallet-cli/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'kwallet-cli'
+pkgname=kwallet-cli
+version=3.02
+revision=1
+archs="i686* x86_64*"
```
Comment (Chocimier) [inline srcpkgs/kwallet-cli/template:5]:
```
post_patch() {
	vsed -i GNUmakefile -e "s|-I/usr/include|-I${XBPS_CROSS_BASE}/usr/include|g"
}
```

fixes cross build

File: srcpkgs/kwallet-cli/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+wrksrc=kwalletcli
+build_style=gnu-makefile
+make_build_args="KDE_VER=5"
+hostmakedepends="pkg-config"
+makedepends="kwallet-devel qt5-devel"
+depends="kwallet qt5 mksh"
```
Comment (Chocimier) [inline srcpkgs/kwallet-cli/template:11]: kwallet and qt5 is detected automatically:

=> kwallet-cli-3.02_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libKF5Wallet.so.5 <-> kwallet>=5.26.0_1
   SONAME: libQt5Widgets.so.5 <-> qt5-widgets>=5.15.0_1

File: srcpkgs/kwallet-cli/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+short_desc="Command-line interface to the KDE Wallet"
+maintainer="Louis Dupré Bertoni <contact@louis.xyz>"
+license="custom:MirOS, LGPL-3.0-or-later"
+homepage="https://www.mirbsd.org/kwalletcli.htm"
+distfiles="http://www.mirbsd.org/MirOS/dist/hosted/kwalletcli/kwalletcli-${version}.tar.gz"
+checksum=f6bebf802d6d575b0871c01dc1cbcc0f182d26cdce28449da426a503a257d216
+
+pre_install() {
+	vmkdir /usr/bin
```
Comment (Chocimier) [inline srcpkgs/kwallet-cli/template:20]: plaese drop initial slashes

File: srcpkgs/kwallet-cli/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/kwallet-cli/template:5]: Once cross build is fixed, no need to limit archs.

**Comment by DBLouis** [comment]
It should be good now

**Pull request merged**